Warp.WebApp/Data/Redis/KeydbStorage.cs

```diff
@@ -33,23 +33,33 @@ public void Remove<T>(string key)
     }
 
 
-    public async Task Set<T>(string key, T value, TimeSpan expiresIn)
+    public async Task Set<T>(string key, T value, TimeSpan expiresIn, CancellationToken cancellationToken = default)
```

Comment:
Better move `default` declarations from here up to controller. It is more consistent IMO